### PR TITLE
Implement Responsive Footer and Additional Links 

### PR DESCRIPTION
**Pull request by TanmayDhobale**
- Description:

This pull request introduces a responsive footer and additional links to enhance the usability and accessibility of the project. The changes made are as follows:

- Responsive Footer: Implemented a responsive footer design that adjusts its layout and content based on screen size. This ensures a consistent user experience across various devices and screen resolutions.

- Additional Links: Added new links to the footer section, providing users with easy access to relevant resources, such as social media profiles, contact information, or additional pages within the project.

-  Changes Made:

Created responsive footer component
Added new links to the footer 

https://github.com/code100x/daily-code/assets/89733575/1631dfb3-7588-4b1f-8472-db7766cac144

### CODE REVIEW

**Comment by TanmayDhobale** [comment]
@hkirat

**Comment by hkirat** [comment]
Please remove package-lock, we're only keeping yarn lock from now on

File: package.json [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-    "lint:fix": "turbo lint:fix",
-    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
-    "prepare": "husky"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^18.6.0",
-    "@commitlint/config-conventional": "^18.6.0",
-    "@repo/eslint-config": "*",
-    "@repo/typescript-config": "*",
-    "autoprefixer": "^10.4.16",
```
Comment (hkirat) [inline package.json:17]: Mujhe kyun toda?

**Comment by TanmayDhobale** [comment]
sry sry  man

**Comment by TanmayDhobale** [comment]
@hkirat  cool now ser ?

**Comment by nimit9** [comment]
Hey @TanmayDhobale , you forgot to add `@icons-pack/react-simple-icons` to the package.json

**Comment by TanmayDhobale** [comment]
> Hey @TanmayDhobale , you forgot to add `@icons-pack/react-simple-icons` to package.json, it's breaking on local



I've added but he marge so fast

**Comment by TanmayDhobale** [comment]
Let me raise pr again for this haha

**Comment by TanmayDhobale** [comment]
> Let me raise pr again for this haha

@nimit9  should i?

**Comment by nimit9** [comment]
Yeah, go ahead

**Pull request merged**